src/components/Link/README.md

```diff
@@ -0,0 +1,192 @@
+<!--GITHUB_BLOCK-->
+
+# Link
+
+<!--/GITHUB_BLOCK-->
+
+```tsx
+import {Link} from '@gravity-ui/uikit';
+```
+
+## Description
+
+Link - is a part of the text, that, when clicked, takes the user to another part of the page, another page inside the service, or to a page of another external service.
+
+The main difference from the [Buttons](/components/uikit/button) is the navigation function. Links more often lead to other pages or open new browser tabs.
+
+## Appearance
+
+There can be three types: `normal`(the usual brown), `primary`(black), `secondary`(gray). it is controlled by the `view` property. It is also possible to display visited state using the `visitable` property.
+
+### Normal
+
+The most familiar and well-established pattern of use for users. It is used to visually highlight an element inside a text, table, and as part of navigation. It is used to navigate to pages inside the service and to external sources, including documentation. Also, they are used on error pages and zero states.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Link view="normal" href="/">Link</Link>
+`}>
+    <UIKit.Link view="normal" href="/">Link</UIKit.Link>
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Link view="normal" href="/">
```

Comment:
Let's use `#` instead of `/`, to avoid page reload on click

Reply:
Done